src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -34,6 +44,11 @@
  * @author Michael Neale
  */
 public class DockerBuilder extends Builder {
+
+    private static final Logger logger = Logger.getLogger(DockerBuilder.class.getName());
+
+    private static final Pattern IMAGE_BUILT_PATTERN = Pattern.compile(".*?Successfully built ([0-9a-f]*).*?", Pattern.DOTALL);
```

Comment:
Do you actually need the `.*?` if you are using `.find` (rather than `.matches`) anyway?